src/client/pythonEnvironments/base/locators/lowLevel/fsWatchingLocator.ts

```diff
@@ -32,13 +32,13 @@ function checkDirWatchable(dirname: string): DirUnwatchableReason {
         names = fs.readdirSync(dirname);
     } catch (err) {
         const exception = err as NodeJS.ErrnoException;
-        traceError('Reading directory to watch failed', exception);
+        traceWarn('Reading directory failed', dirname);
```

Comment:
You need to record the exception here. `traceWarn` is fine just capture the exception line before.
```suggestion
        traceWarn('Reading directory failed', exception);
```